### PR TITLE
Explosives - Add modded defusal kit support

### PR DESCRIPTION
**Pull request by SpicyBagpipes**
**When merged this pull request will:**
- Title, reads off of config property. Last tool in mod without this functionality afaik

### IMPORTANT

- [x] If the contribution affects [the documentation](https://github.com/acemod/ACE3/tree/master/docs), please include your changes in this pull request so the documentation will appear on the [website](https://ace3.acemod.org/).
- [x] [Development Guidelines](https://ace3.acemod.org/wiki/development/) are read, understood and applied.
- [x] Title of this PR uses our standard template `Component - Add|Fix|Improve|Change|Make|Remove {changes}`.

### CODE REVIEW

**Comment by SpicyBagpipes** [comment]
Unsure why pboproject failed, details aren't helpful

**Comment by jonpas** [comment]
@Dahlgren Looks like it's not finding some registry?

File: addons/explosives/functions/fnc_canDefuse.sqf [hunk 1]
```diff
@@ -24,7 +24,7 @@ if (isNull _explosive) exitWith {
     deleteVehicle _target;
     false
 };
-if (vehicle _unit != _unit || {!("ACE_DefusalKit" in (_unit call EFUNC(common,uniqueItems)))}) exitWith {false};
+if (vehicle _unit != _unit || {(_unit call EFUNC(common,uniqueItems)) findAny GVAR(defusalKits) != -1}) exitWith {false};
```
Comment (PabstMirror) [inline addons/explosives/functions/fnc_canDefuse.sqf:27]:
```suggestion
if (vehicle _unit != _unit || {(_unit call EFUNC(common,uniqueItems)) findAny GVAR(defusalKits) == -1}) exitWith {false};
```

**Comment by PabstMirror** [comment]
need to modify ace_explosives_fnc_interactEH as well

**Comment by Dahlgren** [comment]
> @Dahlgren Looks like it's not finding some registry?

it happens if the `pboproject -p` setup call does not precreate the registry entries correctly. Rerunning usually fixes it

**Comment by SpicyBagpipes** [comment]
> need to modify ace_explosives_fnc_interactEH as well

Done

File: addons/explosives/functions/fnc_interactEH.sqf [hunk 2]
```diff
@@ -25,7 +25,7 @@ TRACE_1("Explosives interactEH",_interactionType);
 if (
     _interactionType != 0
     || {vehicle ACE_player != ACE_player}
-    || {!("ACE_DefusalKit" in (ACE_player call EFUNC(common,uniqueItems)))}
+    || {(_unit call EFUNC(common,uniqueItems)) findAny GVAR(defusalKits) == -1}
```
Comment (PabstMirror) [inline addons/explosives/functions/fnc_interactEH.sqf:28]:
```suggestion
    || {(ACE_player call EFUNC(common,uniqueItems)) findAny GVAR(defusalKits) == -1}
```

**Review by PabstMirror — APPROVED** [review]

**Pull request merged**